www/index.html

```diff
@@ -491,7 +491,7 @@ <h3>..and automobiles</h3>
         <h2>Hotels</h2>
         <p>Dundee is well supplied with hotels (budget to luxury), B&amp;Bs (Bed and breakfast guest houses), and rentals.</p>
         <p>Prices are very good compared to elsewhere in the UK.</p>
-        <p>Check back later for recommendations and conference deals.</p>
+        <p>The <a href="https://www.meetdundeecityregion.co.uk/attending/conferences/state-of-the-map-europe">Dundee City Region webpage for SotMEU</a> links to a selection of accommodation options</p>
```

Comment:
We could potentially save visitors a navigation step by hyperlinking directly to DACBRES here; I didn't do that initially because I felt that Dundee City Region is the upstream source, and that people have a better chance to understand and navigate that relationship themselves by visiting DCR first.

```suggestion
        <p>Dundee City Region's <a href="https://dacbres.bzon.uk/event/state-of-the-map-europe-14-15-november-2025/">official accommodation booking partner webpage for SotMEU</a> provides a range of accommodation options</p>
```